### PR TITLE
Add example API for Service Registry from the BOMaaS project

### PR DESCRIPTION
**Pull request by samakinen**
Adds example API definitions for Service Registry API. 

The Service Registry provides methods of sharing metadata related to mobility services between TSPs, MaaS operators and end users.

### CODE REVIEW

**Comment by brylie** [comment]
@maas-alliance/contributors please review. This PR represents a Service Registry and Ticket API. Please leave any feedback/suggestions as comments on this PR.

**Comment by brylie** [comment]
@samakinen not to make your contribution more difficult, but do you think you could open the files as separate PRs? That way, we could review each in their own time, so that merging one doesn't get blocked by the other, for instance.

**Comment by samakinen** [comment]
@brylie no problem, I have split the PR in two

File: specifications/service-registry.yaml [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+    of the service.
+    
+  version: "0.1.0-oas3"
+  title: Service registry
+  contact:
+    name: FLOU
+    url: 'https://www.flou.io/'
+    email: sami.makinen@flou.io
+  license:
+    name: CC BY 4.0
```
Comment (brylie) [inline specifications/service-registry.yaml]: This repository uses an Apache license, which is similar to CC-by for software. Can we change this to use the Apache license?

**Comment by brylie** [comment]
Hey @pomgod, we haven't yet reviewed this PR.

**Comment by nighthawk** [comment]
@brylie to prevent merges that haven't yet been approved you can add "branch protection rules" under the settings for this repo. Would make sense to require at least 1 approving review.

**Pull request merged**